src/GridFS/Bucket.php

```diff
@@ -470,6 +479,9 @@ public function openDownloadStreamByName($filename, array $options = [])
      *  * chunkSizeBytes (integer): The chunk size in bytes. Defaults to the
      *    bucket's chunk size.
      *
+     *  * disableMD5 (boolean): When true, no MD5 sum will be generated for
+     *    the stored file. Defaults to "false".
```

Comment:
These two lines should like be added to the doc block for `uploadFromStream()` further down in the file.